app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,45 @@
+class RelationshipsController < ApplicationController
+  before_action :set_user, only: [:create]
+  before_action :set_relationship, only: [:destroy]
+
+  def create
+    if current_user.follow(@user)
+      flash[:success] = "You are now following #{@user.name}."
+    else
+      flash[:alert] = 'Unable to follow user.'
+    end
+    respond_to do |format|
+      format.html { redirect_to @user }
+      format.turbo_stream
```

Comment:
`.html` vs `. turbo_stream` ở đây khác nhau như nào vậy em

Reply:
respond_to sẽ trả 1 trong 2 dạng là html (tức load trang @user) hoặc trả theo turbo_stream đã được define trước đó tùy theo mong muốn của broswer nguời dùng. html sẽ load lại toàn bộ trang còn turbo_stream sẽ chỉ load những gì mình đã defind trong <action>.turbo_stream ạ. 
Tuy nhiên em vẫn còn một vài thắc mắc như về khi nào nó sẽ chọn load như thế nào ... Em vẫn đang đọc kĩ hơn ạ.